### PR TITLE
Mushfiq

### PR DESCRIPTION
**Pull request by abmushfiq**
[ITP-1770](https://fcodelabs.atlassian.net/browse/ITP-1770)
Initialize project , SignIn Page Created

[ITP-1770]: https://fcodelabs.atlassian.net/browse/ITP-1770?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: dear-diary-app/src/App.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    console.log(name)
+    setName(name)
+  }
+
+  const pageHandler = () => {
+    setIsFirstPage(state=>!state);
+  };
+
+  return <div className="App">
+    {isFirstPage ? <Signin pageHandler={ pageHandler } nameHandler={nameHandler} />:<Main_Page pageHandler={ pageHandler } name={name} />}
```
Comment (shamal-iroshan) [inline dear-diary-app/src/App.js]: use react-router to handle things like switching between pages

File: dear-diary-app/src/components/SigninCard.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+.sign-body{
```
Comment (shamal-iroshan) [inline dear-diary-app/src/components/SigninCard.css]: no need of using CSS like this with MUI use their sx prop or styled to create components with custom styles

File: dear-diary-app/src/components/SigninCard.js [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+    lineHeight: '60px',
+}));
+  
+
+
+function SigninCard({ pageHandler,nameHandler }) {
+
+
+    // these are states for our local value
+    const [g_Name, setG_Name] = useState('');
```
Comment (shamal-iroshan) [inline dear-diary-app/src/components/SigninCard.js]: use camelCase for variable naming

File: dear-diary-app/src/components/SigninCard.js [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+    // console.log(g_Name);
+
+
+    // genarate NickNames in Array
+
+const gName = () => {
```
Comment (shamal-iroshan) [inline dear-diary-app/src/components/SigninCard.js]: move this function to a util folder

File: dear-diary-app/src/pages/Main_Page.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```
Comment (shamal-iroshan) [inline dear-diary-app/src/pages/Main_Page.js]: check the naming of this file

**Review by shamal-iroshan — CHANGES_REQUESTED** [review]
fix commented things

**Comment by abmushfiq** [comment]
i fixed you mentioned isuues can you check ?

**Pull request closed without merging**